### PR TITLE
[TRTLLM Python backend]Fix the output format for client side batching in dynamic batch 

### PR DESCRIPTION
**Pull request by sindhuvahinis**
## Description ##

If the user sends, client side batch then the result is json array. If the user sends single input, then we send back json object. 

Examples:
Request : `"inputs":["The new movie that got Oscar this year"]`
Response : 
```
[
  {
    "generated_text":"The new movie that got Oscar this year is The Shape of Water."
  }
]
```

Request: `"inputs":"The new movie that got Oscar this year"`
Response:
```
{
  "generated_text":"The new movie that got Oscar this year is The Shape of Water."
}
```

### CODE REVIEW

**Comment by lanking520** [comment]
do we want to support client side batching? Maybe just server side is good enough

File: engines/python/setup/djl_python/tensorrt_llm.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def inference(self, inputs: Input) -> Output:
         """
         outputs = Output()
 
-        input_data, input_size, parameters, errors, batch = parse_input(
+        input_data, input_size, parameters, errors, batch, _ = parse_input(
```
Comment (frankfliu) [inline engines/python/setup/djl_python/tensorrt_llm.py:53]: This change break backward compatibility

**Comment by sindhuvahinis** [comment]
> do we want to support client side batching? Maybe just server side is good enough

We already do support. This PR just addresses the output format. In our CI, we also test client side batching. Today's CI would fail because the format is wrong.

File: engines/python/setup/djl_python/tensorrt_llm.py (hunk 1 above)
Comment (sindhuvahinis) [reply engines/python/setup/djl_python/tensorrt_llm.py:53]: Do you mean for people using parse_input in their custom handlers?

Comment (sindhuvahinis) [reply engines/python/setup/djl_python/tensorrt_llm.py:53]: As discussed offline, will introduce another function, not to break the backward compatibility of the users.

**Review by lanking520 — APPROVED** [review]

**Pull request merged**